client-app/components/molecules/in-stock/in-stock.vue

```diff
@@ -0,0 +1,27 @@
+<template>
```

Comment:
This should be atom, not molecule. The markup consists from only HTML tags, there are no other atoms here.